### ISSUE TITLE
Make lifecycle breadcrumbs logged by sentry-android-fragment integration customizable

### ISSUE THREAD

**Issue by mateuszkwiecinski** [getsentry/sentry-java#1734]
## Context
I tried to add `FragmentLifecycleIntegration` to my app, and I quickly reached default breadcrumbs limit. Almost all events came from `FragmentLifecycleIntegration` which decreased number of visible user interactions to last 2-3 screens before crash. 
I could increase breadcrumbs limit, but I concluded I'd prefer see less events logged by `FragmentLifecycleIntegration`.

## Proposal
The simples thing I can suggest is to introduce api like:
```
FragmentLifecycleIntegration(
	application = application,
	loggedLifecycleEvents = setOf(Lifecycle.State.RESUMED, Lifecycle.State.PAUSED),
)
```
which would skip events non-listed in new parameter.

**Comment by marandaneto** [comment]
@mateuszkwiecinski thanks for reporting, such grain fined level makes sense, let's see if people upvote.

a workaround, for now, is using the `BeforeBreadcrumb` callback to filter breadcrumbs out https://docs.sentry.io/platforms/android/enriching-events/breadcrumbs/#customize-breadcrumbs
or obviously, increasing the number of crumbs.

**Comment by romtsn** [comment]
Closes by #2299
